modules/tobira/src/main/resources/OSGI-INF/tobira-api.xml

```diff
@@ -13,4 +13,7 @@
     <reference name="searchService"
                interface="org.opencastproject.search.api.SearchService"
                bind="setSearchService"/>
+    <reference name="seriesService"
```

Comment:
Can we replace the XML with OSGi declarative service annotations?

Reply:
I actually do have some more understanding of the situation, and even a workaround. Talk to me if we want to pursue this. :wink:

Reply:
I don't want to know. I should bother to acquire that knowledge anyway …but  not today.